pkg/streamingpromql/planning.go

```diff
@@ -494,6 +495,31 @@ func (p *QueryPlanner) nodeFromExpr(expr parser.Expr) (planning.Node, error) {
 			return nil, compat.NewNotSupportedError(fmt.Sprintf("'%s' function", expr.Func.Name))
 		}
 
+		if fnc == functions.FUNCTION_INFO {
+			if len(expr.Args) == 1 {
+				infoExpr, err := parser.ParseExpr("target_info")
+				if err != nil {
+					return nil, err
+				}
+				expr.Args = append(expr.Args, infoExpr)
+			} else if len(expr.Args) == 2 {
+				dataLabelMatchersExpr, ok := expr.Args[1].(*parser.VectorSelector)
+				if !ok {
+					return nil, fmt.Errorf("expected second argument to 'info' function to be a VectorSelector, got %T", expr.Args[1])
+				}
```

Comment:
We'll need to add a special case to CSE to ensure it doesn't replace the second argument to `info` with a deduplicated expression.